jmclient/test/commontest.py

```diff
@@ -131,7 +135,7 @@ def query_utxo_set(self, txouts, includeconfs=False):
             result.append(result_dict)        
         return result
 
-    def estimate_fee_per_kb(self, N):
+    def estimate_fee_per_kb(self, N: int) -> int:
```

Comment:
nit in 68352d047a636496ec5272c41b4d168ecf481c40:

Same thing about `N` variable name.